protobuf-c.spec

```diff
@@ -107,6 +128,7 @@ rm -f $RPM_BUILD_ROOT/%{_libdir}/libprotobuf-c.la
 * Wed Oct 02 2019 John E. Malmberg <john.e.malmberg@intel.com> - 1.3.1-1
 - new upstream release
 - Fix most SUSE rpmlint issues
+- protoc needs to be disabled for el7 targets
```

Comment:
Maybe expand on why so that we don't have to go through this exercise in the future.

And it probably is appropriate to restrict the disablement on EL to just 7.  EL8 might be just fine.  If it's not we should find out by the build breaking, not by us remembering to re-enable it when it comes time to test EL8.